File: data/python/COUNT_TRAILING_ZEROES_FACTORIAL_NUMBER_TRANSLATED.java
====================
public static int f_gold ( int n ) {
  int count = 0 ;
  int i = 5 ;
  while ( ( n / i >= 1 ) && ( n / i <= 5 ) ) {
    count += ( int ) ( n / i ) ;
    i *= 5 ;
  }
  return ( int ) count ;
}

File: data/java/COUNT_TRAILING_ZEROES_FACTORIAL_NUMBER_TRANSLATED.py
import sys

def f_gold ( n ) :
    count = 0
    for i in range ( 5 , n / i >= 1 , 1 , 5 ) :
        count += n / i
    return count

if __name__ == '__main__':
    param = [
    (9,),
    (43,),
    (50,),
    (32,),
    (37,),
    (51,),
    (33,),
    (49,),
    (1,),
    (75,)
        ]
    n_success = 0
    print("class,mutant,test_index,result")
    for i, parameters_set in enumerate(param):
        try:
            print("COUNT_TRAILING_ZEROES_FACTORIAL_NUMBER," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("COUNT_TRAILING_ZEROES_FACTORIAL_NUMBER," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
